packages/globals/src/globals.ts

```diff
@@ -1,19 +1,43 @@
-import { KeyboardLayout } from './keyboard-layout';
-
 /* eslint-disable @typescript-eslint/no-explicit-any */
-type ActionType = "interaction" | "check";
+import { KeyboardLayout } from "./keyboard-layout";
+
+export type ActionType = "interaction" | "check";
```

Comment:
For consistency, this should probably be

```ts
export type InteractionType = "action" | "check"
```

Reply:
It was changed here https://github.com/thefrontside/interactors/pull/166

Reply:
Maybe we can do this as a separate PR?

Reply:
In that case it's easier to do it in this PR